### ISSUE TITLE
Console.warn error on reactor.get()

### ISSUE THREAD

**Issue by AiNoKame** [optimizely/nuclear-js#23]
(nuclear-js version 0.5.0-rc4 from npm)

I'm currently testing my nuclear-js stores via mocha, and I keep seeing this output whenever I call reactor.get() using any given KeyPath:

```
iterable.length has been deprecated, use iterable.size or iterable.count(). This warning will become a silent error in a future version. Error
    at Record.Object.defineProperty.get (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/nuclear-js/dist/nuclear.js:1627:20)
    at isArray (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/nuclear-js/dist/nuclear.js:6848:64)
    at Function.module.exports.isKeyPath (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/nuclear-js/dist/nuclear.js:735:7)
    at evaluate (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/nuclear-js/dist/nuclear.js:768:16)
    at Reactor.get (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/nuclear-js/dist/nuclear.js:194:13)
    at getConsolidatedAppState (/Users/rod/Documents/GitHub/ripple-connect-client-app/app/scripts/shared/NuclearTestUtils.js:32:36)
    at Context.setUp (/Users/rod/Documents/GitHub/ripple-connect-client-app/app/scripts/shared/NuclearTestUtils.js:60:26)
    at callFn (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/mocha/lib/runnable.js:266:21)
    at Hook.Runnable.run (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/mocha/lib/runnable.js:259:7)
    at next (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/mocha/lib/runner.js:271:10)
    at Object._onImmediate (/Users/rod/Documents/GitHub/ripple-connect-client-app/node_modules/mocha/lib/runner.js:292:5)
    at processImmediate [as _immediateCallback] (timers.js:345:15)
```

NuclearTestUtils.js:32:36 is the reactor.get().

Looking at logs, this seems to happen when an Immutable Record (a Getter) is passed as an argument into the anonymous function whose return value is assigned to isArray on line 6848. When you try to get the length of the Immutable Record, the error on line 1627 gets thrown.

**Comment by jordangarcia** [comment]
Is this an actual error or just a warning?

ImmutableJS has deprecated `list.length` in favor of `list.size`.

**Comment by AiNoKame** [comment]
This is a warning, but it happens consistently at any invocation of reactor.get(). The warning will always log before the console.log when running this code in mocha:

```
var reactor = require('nuclear-js').Reactor;
var store = require('nuclear-js').Store;

var fooStore = store({
  getInitialState: function() {
    return true;
  }
});

var testReactor = reactor({
  stores: {
    foo: fooStore
  }
});

console.log('post-console.warn', testReactor.get('foo'));
```

**Comment by jordangarcia** [comment]
I will make sure this is fixed in 0.6.0. 

Thanks for bringing it up!
